docs/content/SRS/SRS.md

```diff
@@ -27,35 +27,103 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
 
 ### 2.1 Client
 
-[Insert your content here.]
+Description: The client base for the tinymorph project includes organizations and institutions that benefit from advanced text-editing solutions such as engineering firms, academic institutions, and publishing companies. These clients provide essential resources and guidance for the project.
+
+Role: Responsible for defining the project's scope and deliverables, ensuring funding, and facilitating alignments with broader academic or commercial goals. They play a vital role in strategic decision-making and project validation.
 
 ### 2.2 Customer
 
-[Insert your content here.]
+Description:  The primary customers are professional writers and engineers who require sophisticated tools for technical writing, documentation, and publishing. This group can include freelancers who often juggle various projects, technical writers in engineering fields  who need to document intricate concepts, and content creators in technical disciplines who require robust features for complex content creation.
```

Comment:
idk about technical writing and documentation.

For creative writing it has wider range for dataset. Technical writing are usually solution per-basis.

Reply:
Since we are not targeting cookiecutter writing (in this case documentation, marketing blogs), we might want to target writers, engineers and freelance (though I think freelance are umbrella term for both writers and engineer, as you can do freelance work for engineering)

Reply:
Remove the focus of technical writing and documentation. I put more focus on groups such as engineers and creative writers.